src/ui/options.js

```diff
@@ -76,7 +76,7 @@
       if (f) f.remove();
       f = document.createElement("iframe");
       f.id = id;
-      f.srcdoc = `<a download="noscript_data.txt" target="_blank">NoScript Export</a>`;
+      f.srcdoc = `<a download="noscript_data.json" target="_blank">NoScript Export</a>`;
```

Comment:
BTW, I completely dislike when DOM nodes are created with the ways like `innerHTML` using hardcoded HTML. IMHO they should always be created using DOM API.